app/components/Settings/Settings.jsx

```diff
@@ -231,21 +253,21 @@ class Settings extends React.Component {
 
         return (
             <div className={this.props.deprecated ? "" : "grid-block page-layout"}>
```

Comment:
To remove the blue border you should change this to `tab-layout`

`<div className={this.props.deprecated ? "" : "grid-block tab-layout"}>`

Doesn't seem to break any of your CSS as far as I can see when testing it.